test/Libraries/PackageManagerTests/PackageManagerClientTests.cs

```diff
@@ -63,57 +63,6 @@ public void ListAll_ReturnsEmptyListWhenQueryThrowsException()
 
         #endregion
 
-        #region DownloadPackageHeader
-
-        [Test]
-        public void DownloadPackageHeader_SucceedsForValidPackage()
-        {
-            var id = "1";
-
-            // Returned content
-            var mp = new ResponseWithContentBody<PackageHeader>
```

Comment:
@sm6srw where do you see PackageHeader obsolete?

Reply:
PackageHeader is still there but the PackageHeaderDownload method is obsoleted. Only used by these two tests.